.github/workflows/rust.yml

```diff
@@ -30,10 +30,7 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: clippy
-        # Use one of our supported targets to lint all crates including
-        # the target-specific `ndk` in one go.
-        # This assumes our host-tools (cargo-apk and dependencies)
-        # also compile cleanly under this target.
+        # Use a cross-compilation target (that's typical for Android) to lint everything
```

Comment:
Can you clean this up in the inverse way in the `cargo-apk` dir, since it specifically does not have to run on the target arch anymore, but on a (set of) host(s)?

OTOH that host might as well be an Android phone; linting for all the triples is fine by me but the comment is dated.

Reply:
Right, it'd make sense in `cargo-apk` to lint with host targets, since it's a tool that runs on the host. It should be easy to follow up on iterating / cleaning up the CI in any of the standalone repos.

My first changes were just focused on making minimal CI updates that could ensure everything was building with green lights in the standalone repos.

Reply:
Mostly the comment that's off now, I'm fine having "android" itself as a host if someone wants to build APKs from Android for Android (Termux anyone?).